src/pal/tests/palsuite/c_runtime/vprintf/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 2.8.12.2)
 
-add_subdirectory(test1)
+# This test fails to build on ARM
+#add_subdirectory(test1)
```

Comment:
Additionally, the test is against the C standard anyway, there is no guarantee that va_list will be an array type, this is ABI dependent.